waspc/src/Wasp/Generator/ServerGenerator/JsImport.hs

```diff
@@ -0,0 +1,30 @@
+module Wasp.Generator.ServerGenerator.JsImport where
+
+import Data.Maybe (fromJust)
+import qualified StrongPath as SP
+import qualified Wasp.AppSpec.ExtImport as EI
+import Wasp.Generator.JsImport (RelDirToExternalCodeDir, mkJsImportGetterFromExtSrcDir)
+import Wasp.Generator.ServerGenerator.ExternalCodeGenerator (extServerCodeDirInServerSrcDir)
+import Wasp.JsImport
+  ( JsImport,
+    JsImportIdentifier,
+    JsImportStatement,
+  )
+import qualified Wasp.JsImport as JI
+
+-- | Wrapper function to avoid needing to import getJsImportStmtAndIdentifier from Wasp.JsImport
+--  in most cases.
```

Comment:
Hm this comment confuses me, I would just remove it, I think this is clear from the code.